packages/lesswrong/lib/collections/comments/callbacks.js

```diff
@@ -378,4 +402,4 @@ async function updateTopLevelCommentLastCommentedAt (comment) {
   Comments.update({ _id: comment.topLevelCommentId }, { $set: {lastSubthreadActivity: new Date()}})
   return comment;
 }
-addCallback("comments.new.after", updateTopLevelCommentLastCommentedAt)
\ No newline at end of file
+addCallback("comment.create.after", updateTopLevelCommentLastCommentedAt)
```

Comment:
What does this change do?

Reply:
Just moves to the newer naming convention.